gateway/1.4/modules/ROOT/pages/flex-gateway-securing-local.adoc

```diff
@@ -0,0 +1,21 @@
+= Securing and Governing APIs in Local Mode
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+For Flex Gateway running in Local Mode, you apply API policies using xref:flex-local-configuration-reference-guide.adoc[local declarative configuration files]. You specify the location of these configuration files when first running Flex Gateway.
+
+See xref:flex-local-reg-run.adoc[] for more information about how to start the gateway.
+
+Refer to the following tutorials for examples on applying policies to APIs in Local Mode:
+
+* xref:flex-local-secure-api-with-basic-auth-policy.adoc[]
+* xref:flex-local-secure-api-with-auto-policy.adoc[]
+
+For Flex Gateway running in Connected Mode, refer to xref:flex-gateway-securing-conn.adoc[].
```

Comment:
Since this is the other use case (not supporting the current process), suggest putting it in See Also.

Reply:
Actually I don't think it needs to be there at all. Removing it....